MyPy complains about imported modules with no type hints or library stubs

Output of `mypy gcalcli/details.py`:

```
gcalcli/utils.py:8: error: Skipping analyzing "parsedatetime.parsedatetime": found module but no type hints or library stubs
gcalcli/utils.py:8: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
Found 1 error in 1 file (checked 1 source file)
```

Solution is to add the following `mypy.ini`. This will result in MyPy ignoring any imports that don't have type hints or library stubs. 
```
[mypy]
ignore_missing_imports = True
```

While the MyPy documentation suggests ignoring these one-by-one instead, I have found that provides limited to nonexistent benefits and increases the hassle of using MyPy.

--- a/gcalcli/gcal.py
+++ b/gcalcli/gcal.py
@@ -44,6 +44,7 @@ EventTitle = namedtuple('EventTitle', ['title', 'color'])
 
 CONFERENCE_DATA_VERSION = 1
 
+
 class GoogleCalendarInterface:
 
     cache = {}

--- a/mypy.ini
+++ b/mypy.ini
@@ -1,0 +1,2 @@
+[mypy]
+ignore_missing_imports = True

